Game/Src/TestEnemyMovement.gd

```diff
@@ -241,14 +247,14 @@ func lightning():
 		closest_enemy.die()
 
 func slippery_hands():
```

Comment:
Just a note while I read the code: At one point it looks like we should refactor the code so that these functions that are actions taken by the sword/player are not in a scene named "TestEnemyMovement". But it is not clear to me how to structure that for now

Reply:
Definitely, Ideally, these would be individual nodes that are instanced onto the buttons :)